app/views/choose_an_option.scala.html

```diff
@@ -15,58 +15,54 @@
  *@
 
 @import config.ApplicationConfig
-@(fileStatus: FileUploadStatus.Value, formattedDate: Option[String])(implicit request: Request[_], messages: Messages, appConfig: ApplicationConfig)
-
-@import uk.gov.hmrc.play.views.html.helpers._
-@import views.html.helpers._
 @import models.FileUploadStatus._
 
-@ras_main(title = messages("chooseAnOption.page.title", messages("filestatus." + fileStatus))) {
+@this(govuk_wrapper: views.html.govuk_wrapper)
+
+@(fileStatus: FileUploadStatus.Value, formattedDate: Option[String])(implicit request: Request[_], messages: Messages, appConfig: ApplicationConfig)
+
+@govuk_wrapper(pageTitle = messages("chooseAnOption.page.title", messages("filestatus." + fileStatus))) {
 
-    <header class="page-header">
-        <h1 class="heading-xlarge">@messages("chooseAnOption.page.header")</h1>
-    </header>
+    <h1 class="govuk-heading-xl">@messages("chooseAnOption.page.header")</h1>
```

Comment:
Could this use _@hmrcPageHeading_ instead? Think i've seen this in a few over places which uses html instead of the the helper, unless there is a reason for it not to be used.

Reply:
I'll re-review the instances where I have/haven't but for the most part _@hmrcPageHeading_ adds additional padding/margin to the header than the regular _govuk-heading-xl_ class does and sometimes this didn't blend well with other elements which may have used top-padding/margin.

Reply:
Sounds good to me if it comes down to the padding/margin and was intentional.